### PR TITLE
Add Dims bio for 2020 steering election

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: events/elections/2020/davanumsrinivas.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+We are currently optimized for large-ish teams with full time contributors and i am keen on improving participation of individuals who are occasional contributors but with real world needs.
+
+Build better bridges with the broader ecosystems and other open source projects and teach/learn from each other.  
+
+## Where I work
+
+VMware
+
+[@dims]: https://github.com/dims
```
Comment (krmayankk) [inline events/elections/2020/davanumsrinivas.md:37]: i was going to add mine , when i saw your PR https://github.com/kubernetes/community/blob/master/events/elections/2020/biotemplate.md .  Its different from what is in the template or is that template in the process of being updated ?

**Comment by jdumars** [comment]
/lgtm
/hold 
hold pending election officer reviews

**Review by idvoretskyi — APPROVED** [review]
/lgtm

**Pull request merged**